src/core/scene/a-scene.js

```diff
@@ -285,7 +285,7 @@ export class AScene extends AEntity {
                 self.usedOfferSession = false;
               }
 
-              vrManager.layersEnabled = xrInit.requiredFeatures.indexOf('layers') !== -1;
+              vrManager.layersEnabled = xrInit.requiredFeatures.indexOf('layers') !== -1 || xrInit.optionalFeatures.indexOf('layers') !== -1;
```

Comment:
The fact that `layers` is part of the `optionalFeatures` does not mean that it will be available in the session. The check should use [`enabledFeatures`](https://www.w3.org/TR/webxr/#dom-xrsession-enabledfeatures) to determine this.

Though, sadly, some older WebXR implementations like on the Pico Neo3 (Link) and Pico 4 don't implement this, and `enabledFeatures` is undefined...

Reply:
Thanks. Indeed I see such usage in threejs repo https://github.com/search?q=repo%3Amrdoob%2Fthree.js%20enabledFeatures&type=code
I'll change with the line you suggested.
I hope AVP have xrSession.enabledFeatures properly defined then, otherwise the example won't fallback to the plane, please confirm @dmarcos